### PR TITLE
Default Experiences

### PR DESCRIPTION
**Pull request by cameronking4**
Generating Azure Portal default experiences using Typespec, link to latest documentation. 

Using link rather than copying over content for easier maintenance.

### CODE REVIEW

File: docs/howtos/Azure Portal/default-experiences.md [hunk 1]
```diff
@@ -0,0 +1,30 @@
+# Generating experiences in Azure Portal with TypeSpec
+
+Go to the [TypeSpec guide](https://typespec.io/docs/next/libraries/http/content-types#content-type-negotiation) to learn how to customize a default Azure Portal experience for your resource types using TypeSpec. Go to this documentation link to learn how to apply TypeSpec decorators and see an example to brand your generated experience.
```
[the remark this answers is not in the document]
Comment (cameronking4) [reply docs/howtos/Azure Portal/default-experiences.md]: Fixed thanks.

**Review by yejee94 — APPROVED** [review]

File: docs/howtos/Azure Portal/default-experiences.md [hunk 2]
```diff
@@ -0,0 +1,32 @@
+# Generating experiences in Azure Portal with TypeSpec
+
+Go to the [TypeSpec guide](https://eng.ms/docs/products/azure-portal-framework-ibizafx/declarative/typespec) to learn how to customize your Azure Portal experience, generated by default for your ARM resource type. Azure Resource Providers can apply TypeSpec decorators to generate and brand Azure Portal experiences.
```
Comment (yejee94) [inline docs/howtos/Azure Portal/default-experiences.md:3]: nit: bit confusing for me :)

**Review by markcowl — APPROVED** [review]

**Review by markcowl — CHANGES_REQUESTED** [review]
Need to run `pnpm format` to clear the formatting issue

**Comment by markcowl** [comment]
closing in favor off: https://github.com/Azure/typespec-azure/pull/1953

**Pull request closed without merging**